pyproject.toml

```diff
@@ -26,6 +26,8 @@ dependencies = [
     "python-dateutil",
     "paramiko",
     "filelock",
+    "dpres-mets-builder @ git+https://github.com/Digital-Preservation-Finland/dpres-mets-builder.git@v1.0.0",
+    "dpres-siptools-ng @ git+https://github.com/Digital-Preservation-Finland/dpres-siptools-ng.git@v1.0.0",
```

Comment:
Hmm... doesn't this effectively pin them to those versions and install locations too? I think it would be better to just list them without the URLs which allows them to be installed from anywhere (e.g. wheel) and use newer versions for them too.  Of course having ">=1.0.0" for both might also make sense, but I don't know if that really matters so much...

Reply:
I added the Git URLs to make sure they're installed when passari is installed with `pip install .`.  I could add the URLs without the version number to a separate requirements file (e.g. a `requirements-git.txt`). Or do you think we shouldn't specify the URLs in either `pyproject.toml` or any requirements file in this repo?

We'll add the siptools requirements to passari-compose's `requirements-siptools.txt` but I was thinking they should be added here too in case passari is installed separately.

Reply:
I think it's good the way you did it now in the latest commit (13184c7): No version numbers or URLs in pyproject.toml and then listing those urls in a separate file. That's exactly how I think it should be so that the deps can be instaled from other sources too (e.g. wheel).